libfreerdp/codec/rfx.c

```diff
@@ -162,15 +163,11 @@ RFX_TILE* rfx_decoder_tile_new()
 {
 	RFX_TILE* tile = NULL;
 
-	tile = (RFX_TILE*) malloc(sizeof(RFX_TILE));
+	if (!(tile = (RFX_TILE*) calloc(1, sizeof(RFX_TILE))))
+		return NULL;
 
-	if (tile)
-	{
-		ZeroMemory(tile, sizeof(RFX_TILE));
-
-		tile->data = (BYTE*) malloc(4096 * 4); /* 64x64 * 4 */
-		tile->allocated = TRUE;
```

Comment:
@akallabeth I don't see where that would be. Maybe you've looked at the negative/red/old part of the diff ?